cpp/src/ray/test/wrap_test.cc

```diff
@@ -0,0 +1,553 @@
+
+#include <gtest/gtest.h>
+#include <ray/api.h>
+#include <ray/api/serializer.h>
+#include <iostream>
+
+using namespace ray::api;
+
+TEST(ray_marshall, type_bool) {
```

Comment:
```suggestion
TEST(SerializationTest, TestBool) {
```
Actually, I'd suggest merging multiple types in one test case (e.g., all primitive types). As there is some duplicated code in this file.